### PR TITLE
[vcpkg baseline][many ports] Fix regressions

### PR DESCRIPTION
**Pull request by JackBoosY**
- libxpm:mutilple-triplets
```
/bin/sh: line 1: xgettext: command not found
```
Add option `--with-gettext` and set it to `no` to disable find gettext since it doesn't provides requireed tool `xgettext` now.

- PASSING, REMOVE FROM FAIL LIST: atk:x64-windows-static 

Can't be reproduced now, this item was introduced by PR https://github.com/microsoft/vcpkg/pull/25937.
cc @Neumann-A @BillyONeal  to double confirm this.

- PASSING, REMOVE FROM FAIL LIST: llfio:arm64-windows

Can't be reproduced now, this item was introduced by PR https://github.com/microsoft/vcpkg/pull/25560.
And the upstream fixed this regression in PR https://github.com/ned14/quickcpplib/pull/33.
~~Update ned14-internal-quickcpplib to merge this upstream modification to avoid this regression happen again.~~
cc @@BurningEnlightenment to confirm this.
Port `ned14-internal-quickcpplib` was updated in PR https://github.com/microsoft/vcpkg/pull/26724.

All regressions are related to https://dev.azure.com/vcpkg/public/_build/results?buildId=77774&view=results

### CODE REVIEW

**Comment by BurningEnlightenment** [comment]
> PASSING, REMOVE FROM FAIL LIST: llfio:arm64-windows

The updated `ned14-internal-quickcpplib` has already been included in #26724, but I forgot about the arm64ec issue and didn't update the baseline. Please revert your `ned14-internal-quickcpplib` update, because the version referenced here (`fe9185172db1739ca37d97f8963349d94c701413`) is older than the one I added in #26724 (`1dac5b6bf79b5ec8eb3a75fa75844a30f88b8559`, see https://github.com/ned14/quickcpplib/commits/master)

**Review by BurningEnlightenment — DISMISSED** [review]
Thanks!

**Comment by dg0yt** [comment]
The libxpm changes are invalid. Either you want `xgtettext`, then you need a host dependency on `gettext[tools]`, or you want to disable it, then you can do this right in the patch, not in a combination of patch plus patched-in option.

**Comment by JackBoosY** [comment]
> The libxpm changes are invalid. Either you want `xgtettext`, then you need a host dependency on `gettext[tools]`, or you want to disable it, then you can do this right in the patch, not in a combination of patch plus patched-in option.

Oh I didn't noticed gettext has this feature. Thanks! I will take a look later.

File: ports/libxpm/portfile.cmake [hunk 1]
```diff
@@ -17,11 +17,17 @@ vcpkg_from_gitlab(
 
 set(ENV{ACLOCAL} "aclocal -I \"${CURRENT_INSTALLED_DIR}/share/xorg/aclocal/\"")
 
+if ("gettext" IN_LIST FEATURES)
+    set(EXTRA_OPTIONS --with-gettext=yes)
+else()
+    set(EXTRA_OPTIONS --with-gettext=np)
```
Comment (dg0yt) [inline ports/libxpm/portfile.cmake:23]: `...=no`

**Review by dg0yt — CHANGES_REQUESTED** [review]
Note that `xgettext` is for extracting strings from sources. It shouldn't be needed at all when building a software which uses gettext.

**Pull request merged**